com.woltlab.wcf/templates/recentActivityListItem.tpl

```diff
@@ -5,6 +5,8 @@
 			
 			<div>
 				<div class="containerHeadline">
+					{event name='containerHeadline'}
```

Comment:
What should go in here? The top line is already stuffed with details at this point.

Reply:
Here the same, it would be nice to have an event between `{foreach}` and `{/foreach}`.